### ISSUE TITLE
Allow importing external files into my config file

### ISSUE THREAD

**Issue by mattdipasquale** [puma/puma#916]
See [Puma config.rb require](http://stackoverflow.com/questions/35737583/puma-config-rb-require).

**Comment by evanphx** [comment]
In the future, please put the content of the request into the github issue itself.

**Comment by mattdipasquale** [comment]
OK, I sure will. 👍 I'm glad to be of service. 🙏 I appreciate you for implementing this request, especially so quickly. That's rare. I'm impressed & humbled. Thank you for your awesome contributions to the Ruby community. You are a true legend. :-)

**Comment by evanphx** [comment]
It’s my pleasure.

**Comment by mattdipasquale** [comment]
The `import` method doesn't seem to work for Puma 3.8.2. Please advise.

**Comment by evanphx** [comment]
@mattdipasquale In what doesn't it work?

**Comment by mattdipasquale** [comment]
In `config/puma/development.rb`. The first line is: `import 'base'`, to import `config/puma/base.rb`.

When I (after applying a quick fix for [issue #1265][1]) run `RACK_ENV=development bundle exec puma --debug`, I get the following error output:

```
bundler: failed to load command: puma (/usr/local/bin/puma)
NoMethodError: undefined method `import' for #<Puma::DSL:0x007fc2701bae60>
Did you mean?  port
  config/puma/development.rb:1:in `_load_from'
  /usr/local/lib/ruby/gems/2.4.0/gems/puma-3.8.2/lib/puma/dsl.rb:41:in `instance_eval'
  /usr/local/lib/ruby/gems/2.4.0/gems/puma-3.8.2/lib/puma/dsl.rb:41:in `_load_from'
  /usr/local/lib/ruby/gems/2.4.0/gems/puma-3.8.2/lib/puma/configuration.rb:204:in `block in load'
  /usr/local/lib/ruby/gems/2.4.0/gems/puma-3.8.2/lib/puma/configuration.rb:203:in `each'
  /usr/local/lib/ruby/gems/2.4.0/gems/puma-3.8.2/lib/puma/configuration.rb:203:in `load'
  /usr/local/lib/ruby/gems/2.4.0/gems/puma-3.8.2/lib/puma/launcher.rb:59:in `initialize'
  /usr/local/lib/ruby/gems/2.4.0/gems/puma-3.8.2/lib/puma/cli.rb:68:in `new'
  /usr/local/lib/ruby/gems/2.4.0/gems/puma-3.8.2/lib/puma/cli.rb:68:in `initialize'
  /usr/local/lib/ruby/gems/2.4.0/gems/puma-3.8.2/bin/puma:8:in `new'
  /usr/local/lib/ruby/gems/2.4.0/gems/puma-3.8.2/bin/puma:8:in `<top (required)>'
  /usr/local/bin/puma:22:in `load'
  /usr/local/bin/puma:22:in `<top (required)>'
```

I looked, and I don't see the `import` method defined in [`lib/puma/dsl.rb`][2] anymore.


  [1]: https://github.com/puma/puma/issues/1265#issuecomment-292755082
  [2]: https://github.com/puma/puma/blob/master/lib/puma/dsl.rb
